### ISSUE TITLE
White interface when starting

### ISSUE THREAD

**Issue by Jorisweijdom** [benkuper/Chataigne#71]
**Environment**
 - OS: Windos 10
 - Version1.7.4

**Describe the bug**
When starting Chataigne I see the program window the top menu bar but the rest of the interface is white.
I cannot close Chataigne now any more and need to do this with TaskManager.
If I delete the 'Chataigne.settings' -file and reboot Chataigne it does start perfectly but of course all my settings are gone and I get the welcome splashscreen.

**To Reproduce**
1. 
2. 
3.
4. 

**Expected behavior**

**Additional context**
![Capture](https://user-images.githubusercontent.com/44802114/95001316-f495fb00-05c8-11eb-89f0-ab74df275fb5.JPG)

**Comment by benkuper** [comment]
That's a very odd bug, never seen or heard of... !
I would start by checking on the graphics card settings. can you try forcing using the dedicated GPU ?

**Comment by benkuper** [comment]
Also, can you try launching Chataigne first, and then opening your file ?

**Comment by Jorisweijdom** [comment]
Hi Ben,
also have the problem when launching Chataigne fresh.
Have tried to start Chataigne with 'Run as administrator', same problem.
Have tried to run with compatibility settings (Win 8), same problem.
Have set the Graphics Settings in Windows to use the  'High performance' GPU (NVIDIA Geforce GTX 1070), same problem.
Have set CHataigne in the NVDIA 'Program settings' list, but dont know what to specifically put on or off... suggestions?

Cheers.
And by the way, thank you for making this already fantastic software!
Joris

**Comment by Jorisweijdom** [comment]
Only when I delete the config file it shows the interface ...

**Comment by benkuper** [comment]
that's super weird...

**Comment by benkuper** [comment]
Ok after checking your crash file, it is more and more clear that it's an nvidia driver related problem, the dump traces back to nvoglv64.dll, which is a known file that has already caused crashed, black screen and whatnot with different Nvidia driver versions on different apps.
Can you check downgrading / upgrading your driver version, removing it altogether, and other tests like that to see if there is one situation that could make it work ?

**Comment by Jorisweijdom** [comment]
Hi Ben,
have been installing other NVIDIA drivers with no change. 

However, I do think you are right about the problem. I now figured out it is about my second screen (using a laptop) with an DP cable being connected through an active adapter to my HDMI port in the laptop. I now noticed that if I start Chataigne on my main laptop screen I have no problem, only when I move it to my second screen and close it the problem with the white screen starts when Chatagine starts on the second screen. This also explains why deleting the config file was a 'solution' because it seems that Chataigne would then start at the main screen first. So as a temporary solution I now make sure I move Chataigne back to the main screen before I close...

I'm still not sure why Chataigne is extra sensitive to this issue and not other software I use so far. But indeed it seems more of a Laptop HDMI port + NVIDIA communication error that anything else.
